lib/Runtime/ByteCode/FuncInfo.h

```diff
@@ -95,6 +96,13 @@ class FuncInfo
     Js::RegSlot outArgsCurrentExpr; // max number of out args accumulated in the current nested expression
     uint        innerScopeCount;
     uint        currentInnerScopeIndex;
+    struct OptionalChainInfo {
```

Comment:
Is this just a struct containing a single pointer?

I may be rusty or missing something but is the struct necessary, could we just use its contents?

Reply:
This is a leftover from 2cde756c664a6aa79ea542e67dbe32b94b4921f6
I'll remove that